trinity/plugins/builtin/tx_pool/plugin.py

```diff
@@ -61,7 +64,10 @@ def configure_parser(self, arg_parser: ArgumentParser, subparser: _SubParsersAct
 
     def handle_event(self, activation_event: BaseEvent) -> None:
         if isinstance(activation_event, TrinityStartupEvent):
-            self.is_enabled = activation_event.args.tx_pool
+            light_mode = activation_event.args.sync_mode == SYNC_LIGHT
+            self.is_enabled = activation_event.args.tx_pool and not light_mode
+            if light_mode:
+                self.logger.warning('The transaction pool is not yet available in light mode')
```

Comment:
I think this should be an error and cause the process to terminate.  I'd rather the user re-run the command without the flag so there is no ambiguity.